src/modules/plotting.py

```diff
@@ -0,0 +1,91 @@
+import json
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+
+class HarryPlotter:
+    def __init__(self, data_directory, plot_path):
+        """
+        Initialize the RLMetricPlotter.
```

Comment:
The class should process only one file at a time.
```suggestion
    def __init__(self, metrics_path: str, plot_path: str):
```